setup.py

```diff
@@ -50,4 +50,5 @@
         'Programming Language :: Python :: 3.7',
     ],
     namespace_packages=['sphinxcontrib'],
+    entry_points = {'console_scripts': 'oas2rst = sphinxcontrib.openapi.cmdline:main'},
```

Comment:
I'm not quite sure this is going to be a quite popular functionality. @olivier-heurtier what do you think if we hide it behind the following call:

```
python -m sphinxcontrib.openapi [ARGS...]
```

i.e. the idea is to do not produce `oas2rst` console script?